licensecheck.pl

```diff
@@ -0,0 +1,551 @@
+#!/usr/bin/perl -w
```

Comment:
It looks like this was copy-and-pasted and has curious indentation. I have no way of determining whether this is maintained somewhere and making sure our version is up-to-date. Adding this file to this repository means we have to maintain it going forward manually.

With other linting/testing things, we install a package/library and then use that. Can we do that with license checking, too?

Reply:
`licencecheck.pl` is now available as a separate package in most distributions.  e.g. https://packages.debian.org/sid/licensecheck . iirc it was part of `devscripts` on Debian in jessie and earlier, but jessie-backports now has the licensecheck package and updated scripts.  Unfortunately it is only a [separate package](https://packages.ubuntu.com/zesty/licensecheck) in Ubuntu zesty and onwards.

Reply:
@bllag Can you install the package rather than include the file? This might help:

https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-on-Standard-Infrastructure

Reply:
I believe what @jayvdb is saying is that I cannot simply install the licensecheck package in the Travis tests, because Travis doesn't yet support Ubuntu Zesty or newer. @jayvdb can you confirm that?

If this PR gets merged, once Travis supports Ubuntu Zesty+, I would be more happy to rip out the bundled `licensecheck.pl` script and modify the `.travis.yml` file to install it from the package repository.

Reply:
I am pretty sure you can use apt to install the jessie-backport onto a trusty node.
If not, it is easy to backport , or install from source; maybe even a single wget could do it.  It is a tiny chunk of code without many dependencies.  Definitely, no need to put the source into this repository.